src/pwr.ps1

```diff
@@ -1074,7 +1194,14 @@ $PwrConfigPath, $PwrConfig = Get-PwrConfig
 $PwrLock = "$PwrPath\pwr.lock"
 $PwrAuths = Get-Content "$PwrPath\auths.json" -ErrorAction SilentlyContinue | ConvertFrom-Json | ConvertTo-HashTable
 $PwrRepositories = Get-PwrRepositories
-Get-PwrPackages
+
+Lock-PwrLock
+try {
+	Get-PwrPackages
```

Comment:
I think this may fix the issues we've seen with multiple, overlapping fetches from invoking `pwr`, since this function tries to update out-of-date packages.